File: conditionals/time_of_day.py
# Elizabeth Gutierrez, Time of day in Python

import time

num1 = "Goodmorning"
num2 = "Good afternoon"
num3 = "Good evening"
i=int(time.time())+60  # setting a checkpoint 60s in the future
if num1 < num2:
    print("Good morning!")
    print("Good afternoon!")
    print("Good evening!")
    
def time_to_min(hour, minute):
    return hour*60+minute

timenow = time.localtime()

morning = timenow.tm_hour
afternoon = timenow.tm_hour
evening = timenow.tm_hour

if evening >= morning and evening <= afternoon:
    print("Good morning!\n")
elif:
    print("Good morning!\n") 

File: conditionals/time_of_day.c
// Elizabeth Gutierrez, Time of Day in C
#include <stdio.h>
#include <time.h>

int main() {
    time_t t;
    struct tm *tm_info;

    time(&t);
    tm_info = localtime(&t);

    printf("%d\n", tm_info->tm_hour);

    if (tm_info->tm_hour <= 12) {
        printf("Good morning!\n");
    } else if (tm_info->tm_hour <= 18) {
        printf("Good afternoon!\n");
    } else {
        printf("Good evening!\n");
    }

    return 0;
}

